api/registry/registry_integration.go

```diff
@@ -35,14 +35,34 @@ func (d *idm) Init(ctx types.Context, config gofig.Config) error {
 	d.config = config
 	d.ctx = ctx
 	d.used = map[string]int{}
-	ctx.WithField("pathCache", "").Debug("checking volume path cache setting")
+	ctx.WithField(types.ConfigIntegrationVolPathCache, d.pathCache()).
+		WithField("serviceName", ctx.ServiceName()).Info(
+		"empty path for unused volume setting")
+	ctx.WithField(types.ConfigIntegrationVolUnmountIgnoreUsed,
+		d.ignoreUsedCount()).
+		WithField("serviceName", ctx.ServiceName()).Info(
+		"ignore used count on unmount setting")
+	ctx.WithField(types.ConfigIntegrationVolMountPreempt, d.preempt()).
+		WithField("serviceName", ctx.ServiceName()).Info(
+		"pre-emptive mount setting")
+	ctx.WithField(types.ConfigIntegrationVolCreateDisable, d.disableCreate()).
+		WithField("serviceName", ctx.ServiceName()).Info(
+		"volume creation disabled setting")
+	ctx.WithField(types.ConfigIntegrationVolRemoveDisable, d.disableRemove()).
+		WithField("serviceName", ctx.ServiceName()).Info(
+		"volume removal disabled setting")
+
```

Comment:
Hi @clintonskitson,

It appears this section was not updated based on my previous comment. Regardless, there should be no reason to log the service name as it should be in the context. If it's not it should be added before the code gets here.